
# message to set pid constants

uint8 DoF

float32 Kp
float32 Ki
float32 Kd

float32 AcceptableError

float32 Ko
